src/style.css

```diff
@@ -1,10 +1,28 @@
-@import url('https://fonts.googleapis.com/css2?family=Comic+Neue:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bungee+Spice&display=swap');
+
```

Comment:
- [ ] Kindly use the appropriate live demo link in your readme file.

The current one leads to `https://k0ppai.github.io/to-do-list-app/dist`